modules/nw-infw-operator-installing.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-infw-operator_{context}"]
+= Installing Ingress Node Firewall Operator
+
+As a cluster administrator, you can install the Ingress Node Firewall Operator by using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== CLI: Installing the Ingress Node Firewall Operator
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware with nodes that have hardware that support Ingress Node Firewall Operator.
+* Install the OpenShift CLI (`oc`).
+* An account with `cluster-admin` privileges.
+
+.Procedure
+
+. To create the `openshift-ingress-node-firewall-operator` namespace, enter the following command:
```

Comment:
@JoeAldinger namespace should be the one that is being used in YAML i.e openshift-ingress-node-firewall.